app/styles/modules/_input-row.scss

```diff
@@ -144,3 +144,10 @@
     color: $input-row-focus-border-color;
   }
 }
+
+&.unblock-code {
```

Comment:
This dosesn't pass our lint [rules](https://circleci.com/gh/mozilla/fxa-content-server/9225?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). You should also nest this inside the the [input tag](https://github.com/anweshknayak/fxa-content-server/blob/f25b94fa26ba4176ce8150ca47b129fb7feac910/app/styles/modules/_input-row.scss#L25) above.